homeassistant/components/keba/lock.py

```diff
@@ -59,3 +62,36 @@ def update_callback(self) -> None:
     async def async_added_to_hass(self) -> None:
         """Add update callback after being added to hass."""
         self._keba.add_update_listener(self.update_callback)
+
+
+class KebaLockUser(LockEntity):
+    """The entity class for KEBA charging stations switch."""
```

Comment:
Why should it be a lock and not a switch entity?
Also the class documentation specifies it as a switch

Reply:
I copied the already existing KebaLock from above (see line 31) and modified it for the user auth.

Reply:
That's not an answer to my question.
Can you give me a reason why this should be a lock and not a switch?
In my opinion this is a switch, which enables/disbales something

Reply:
Ok, sorry. 
This could also be a switch..  
It authorizes / blocks the charging of a EV connected to the KEBA wallbox.

Reply:
Lock entities should only be used for real locks (like the things that block people from coming in your house), all other use cases should be a switch

Reply:
I can change it, but this would be a braking change for the old implementation?

Reply:
We should deprecate it and migrate it to a new entity then